services/oauth/client.go

```diff
@@ -0,0 +1,26 @@
+package oauth
+
+import (
+	. "google-scraper/helpers"
+
+	"github.com/go-oauth2/oauth2/v4/models"
+	"github.com/google/uuid"
+)
+
+func GenerateOauthClient(userID int64) (client *models.Client, err error) {
+	clientId := uuid.New().String()
+	clientSecret := uuid.New().String()
+
+	client = &models.Client{
+		ID:     clientId,
+		Secret: clientSecret,
+		UserID: IntToString(userID),
```

Comment:
Why is the ID stored as a string? 🤔 Is this column be default provided by the oauth2 library?

Reply:
Yes, this column provided by [Oauth2](https://github.com/go-oauth2/oauth2/blob/master/models/client.go#L8) and its type also string